refinery/tool_manager/tests.py

```diff
@@ -84,3 +89,69 @@ def test_unallowed_http_verbs(self):
         self.assertEqual(
             self.delete_response.data['detail'], 'Method "DELETE" not '
                                                  'allowed.')
+
+
+class ToolDefinitionGenerationTests(TestCase):
+
+    def test_malformed_workflow_annotation(self):
+        pass
+
+    def test_workflow_improperly_annotated(self):
+        with open("tool_manager/test-data/workflow_bad_annot.json", "r") as f:
+            wf_data = json.loads(f.read())
+            self.assertRaises(
+                CommandError, validate_workflow_annotation, wf_data)
+
+    def test_list_workflow_tool_def_generation(self):
```

Comment:
What do you mean by "Doing too many things" 
I see multiple assertions being done, but all are within the scope of `test_list_workflow_tool_def_generation` etc.

I think it would be too much to separate each one into it's own test, no?